db/migrate/20170814213115_add_amount_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddAmountToProducts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :products, :amount, :float
```

Comment:
you could have done a `rename_column :products, :price, :amount`

Reply:
didn't know that was a thing. cool!